utilities.py

```diff
@@ -868,16 +574,29 @@ def generateExternalFunction(pathOpenSimModel, outputDir, pathID,
                           compiler=compiler)
         
     # %% Verification
-    # Run ID with the .osim file and verify that we can get the same torques 
-    # as with the external function.
+
+    # delete previous saved dummy motion if needed
+    if os.path.exists(os.path.join(pathID, "DummyDat.sto")):
+        os.remove(os.path.join(pathID, "DummyDat.sto"))
+
+    # Create a dummy motion for ID
+    DummyData = np.zeros((10, nCoordinates + 1))
+    for coor in range(nCoordinates):
+        DummyData[:, coor + 1] = np.random.rand()+0.5
```

Comment:
@MaartenAfschrift we would want the model to be in contact with the floor so that the contact spheres are loaded. If I replace this line with `DummyData[:, coor + 1] = np.random.rand()/100+0.05` to have values similar to the previously used dummy motion file, then the test fails. Could you please take a look at that? I spent some time trying to debug but I must be missing something. Thanks.

Reply:
That is weird. When I use values of 0.05 (for every coordinate) I get a maximal ID-error of 4.9090995157818895e-09 in your version and the new one in this PR. Could you check it again win this new implementation (wihthout the random values)?

I also noticed that you used -0.05 for the pelvis y coordinate (I guess to ensure contact between the spheres on feet and the ground). I guess that this is not needed the (with extended vertical legs) the threshold value for pelvis coordinate value is around 0.9 for contact with the ground.

Reply:
Sounds good. I am fine removing the rand. I wonder if we could use just use fixed values for all coordinates but different values to make sure there is no problem with the state order (eg 0, 0.01, 0.02, etc instead of 0.05 for all).